onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -513,6 +514,28 @@ def testProfilerWithSessionOptions(self):
                     self.assertTrue(tag in lines[i])
             self.assertTrue(']' in lines[8])
 
+    def testProfilerGetStartTimeNs(self):
+        # Get current nanoseconds
+        start = time.monotonic_ns()
+        x = np.array([[1.0, 2.0], [3.0, 4.0], [5.0, 6.0]], dtype=np.float32)
+        
+        # Get 1st profiling's start time
+        so = onnxrt.SessionOptions()
+        so.enable_profiling = True
+        sess = onnxrt.InferenceSession(get_name("mul_1.onnx"), sess_options=so)
+        sess.run([], {'X': x})
+        start_time_1 = sess.get_profiling_start_time_ns()
```

Comment:
These are duplicated code, move into a  helper?

Reply:
Good catch. Thank you @gwang-msft